src/site/content/en/learn/html/text-basics/index.md

```diff
@@ -3,10 +3,198 @@ title: 'Text basics'
+
+<p lang="fr-FR"> HAL a dit : <q>Je suis désolé &lt;NOM SUPPRIMÉ, RIP&gt;, mais j'ai bien peur de ne pas pouvoir le faire, .</q></ p>
+```
+
+The inline quotation element, `<q>`, adds language-appropriate quotes. The user-agent default styles include open-quote and close-quote generated content:
+
+```css
+q::before {content: open-quote;}
+q::after {content: close-quote;}
+```
+
+The `lang` attribute is included to let the browser know that, while base language of the page was defined as English in the `<html lang="en-US"> opening tag, this paragraph of text is in a different language. This helps voice controls such as Siri, Alexa, and voiceOver use French pronunciation. It also informs the browser what type of quotes to render.
+
+Like `<blockquote>`, the `<q>` element supports the `cite` attribute.
+
+```html
+<p> HAL said, <q cite="https://youtube.com/clip/UgkxSc71fLmjI7tNSgy6o7tZ9GxhSz4S-MNh">I'm sorry &lt;NAME REDACTED, RIP&gt;, but I'm afraid I can't do that, </q></p>
+```
+
+### HTML Entities
+
+You may have noticed the escape sequence or "entity". Because the `<` is used in HTML, you have to escape it using either `&lt;` or a less easy-to-remember encoding `&#x3C;`. There are four reserved entities in HTML: `<`, `>`, `&`, and `"`. Their character references are `&lt`, `&gt`, `&amp;` and `&quot;` respectively.
+
+A few other entities you will often use are `&copy;`  for copyright (©),` &tm;` for Trademark (™), and `&nbsp;` for non-breaking space.
+Non-breaking spaces are useful when you want to include a space between two characters or words while preventing a line break from occurring there.
+There are over 2,000 [named character references](https://html.spec.whatwg.org/multipage/named-characters.html#named-character-references).
+But, if needed, every single character, including emojis, has an encoded equivalent that starts with `&#`.
+
+If you take a look at ToastyMcToastface's workshop review (not included in the code sample above), there are some unusual text characters:
+
+```html
+<blockquote>Learning with Hal and Eve exceeded all of my wildest fantasies. All they did was stick a USB in. They promised that it was a brand new USB, so we know there were no viruses on it. The Russians had nothing to do with it. This has no̶̼͖ţ̘h̝̰̩͈̗i̙̪n͏̩̙͍̱̫̜̟g̢̣ͅ ̗̰͓̲̞̀t͙̀o̟̖͖̹̕ ͓̼͎̝͖̭dó̪̠͕̜ ͍̱͎͚̯̟́w̮̲̹͕͈̟͞ìth̢ ̰̳̯̮͇</blockquote>
+```
+
+The last sentence in this blockquote can also be written as:
+
+```html
+This has no&#x336;&#x33C;&#x356;&tcedil;&#x318;h&#x31D;&#x330;&#x329;&#x348;&#x317;i&#x319;&#x32A;n&#x34F;&#x329;&#x319;&#x34D;&#x331;&#x32B;&#x31C;&#x31F;g&#x322;&#x323;&#x345; &#x317;&#x330;&#x353;&#x332;&#x31E;&#x300;t&#x359;&#x300;o&#x31F;&#x316;&#x356;&#x339;&#x315; &#x353;&#x33C;&#x34E;&#x31D;&#x356;&#x32D;d&oacute;&#x32A;&#x320;&#x355;&#x31C; &#x34D;&#x331;&#x34E;&#x35A;&#x32F;&#x31F;&#x301;w&#x32E;&#x332;&#x339;&#x355;&#x348;&#x31F;&#x35E;&igrave;th&#x322; &#x330;&#x333;&#x32F;&#x32E;&#x347;
```

Comment:
Would it be preferable for this to clip onto multiple lines?

Reply:
yes